common/src/main/scala/uk/ac/wellcome/models/Identifiable.scala

```diff
@@ -8,4 +8,6 @@ trait Identifiable {
   def id: String = canonicalId.getOrElse(
     throw UnidentifiableException()
   )
+  val sourceIdentifier: SourceIdentifier
+  val ontologyType: String
```

Comment:
Id minter needs these to be able to mint ids so enforcing them in the base trait

Reply:
(I’d forgotten these weren’t already there!)